charts/argo-cd/README.md

```diff
@@ -2,7 +2,7 @@
 
 A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 
-Source code can be found [here](https://argo-cd.readthedocs.io/en/stable/)
+Source code can be found [here](https://github.com/argoproj/argo-helm/tree/main/charts/argo-cd)
```

Comment:
The link should probably point to `https://github.com/argoproj/argo-cd/` instead of Helm chart repo you are looking at.